CMakeLists.txt

```diff
@@ -11,6 +11,11 @@ include(GetWebsocket) # ?
 include(GetZmqpp) # For networking between AI, RobotHub and World
 include(BuildLibusb) # USB library for RobotHub
 include(CheckCXXCompilerFlag) # For testing compiler cxx standard support
+include(GetGoogleTest) # For testing
+
+# Recommended by official docs: https://google.github.io/googletest/quickstart-cmake.html
+include(GoogleTest)
+enable_testing()
```

Comment:
I see Rolf requested to remove this line. I removed it and everything still *seems* to work. Could you verify if this is still necessary to have? Because if it is not necessary, it would be best to remove this line.

Reply:
I would not be partial to removing it - the official documentation includes it. Especially if including it doesn't have any adverse effects.